core/src/mender-artifact-download.c

```diff
@@ -0,0 +1,120 @@
+/**
+ * @file      mender-artifact-download.c
+ * @brief     Mender artifact download implementation
+ *
+ * Copyright Northern.tech AS
```

Comment:
Same question as for the other file.

Reply:
This one has Joel's code, that's true. Adding.